azure-pipelines.yml

```diff
@@ -221,60 +222,63 @@ jobs:
       verilator --version
       verible-verilog-lint --version
     displayName: Display environment
-  - bash: ci/scripts/build-chip-verilator.sh earlgrey
+  - bash: ci/scripts/build-chip-verilator.sh englishbreakfast
     displayName: Build simulation with Verilator
   - template: ci/upload-artifacts-template.yml
     parameters:
       includePatterns:
-        - "/hw/top_earlgrey/Vchip_earlgrey_verilator"
+        - "/hw/top_englishbreakfast/Vchip_englishbreakfast_verilator"
 
-- job: chip_englishbreakfast_verilator
-  displayName: Build Verilator simulation of the English Breakfast toplevel design
+- job: execute_fast_verilated_tests
+  displayName: Build and run fast tests on sim_verilator
+  pool: ci-public
+  timeoutInMinutes: 200
   dependsOn: lint
-  condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'))
-  pool:
-    vmImage: ubuntu-18.04
   steps:
   - template: ci/install-package-dependencies.yml
+  - task: DownloadSecureFile@1
+    condition: eq(variables['Build.SourceBranchName'], 'master')
+    name: bazelCacheGcpKey
+    inputs:
+      secureFile: "bazel_cache_gcp_key.json"
+  - bash: echo "##vso[task.setvariable variable=bazelCacheGcpKeyPath]$(bazelCacheGcpKey.secureFilePath)"
+    condition: eq(variables['Build.SourceBranchName'], 'master')
+    displayName: Set the remote cache GCP key path
   - bash: |
-      python3 --version
-      fusesoc --version
-      verilator --version
-      verible-verilog-lint --version
-    displayName: Display environment
-  - bash: ci/scripts/build-chip-verilator.sh englishbreakfast
-    displayName: Build simulation with Verilator
+      export GCP_BAZEL_CACHE_KEY=$(bazelCacheGcpKeyPath)
+      ci/scripts/run-fast-verilator-tests.sh
+    displayName: Build and execute tests
+  # TODO: build and cache the verilator model to avoid building twice
```

Comment:
It looks like you've done it now, should this comment be removed?

Reply:
Not quite - the Verilator model is still being built by both the fast and slow test scripts. I've filed an issue to explain more and track this: #12574. Some work needs to be done to upload and download the correct artifacts.